### PR TITLE
Support AU915 RP1 frequencies

### PR DESCRIPTION
**Pull request by Mandur**
# PR for issue #1730
done with the help of @Ouphi

## What is being addressed

Adding Australian frequencies to upstream

## How is this addressed

- Add Arduino Sample used to manually test (manually tested ABP, OTAA, C2D)
- Add unit tests
- Add implementation

### CODE REVIEW

File: LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWanNetworkSrvModule/Program.cs [hunk 1]
```diff
@@ -32,4 +32,5 @@
 };
 
 var configuration = NetworkServerConfiguration.CreateFromEnvironmentVariables();
+configuration.RunningAsIoTEdgeModule = false;
```
Comment (fbeltrao) [inline LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWanNetworkSrvModule/Program.cs:35]: Is this change intentional?

File: LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Regions/LoRaRegionType.cs [hunk 2]
```diff
@@ -11,6 +11,7 @@ public enum LoRaRegionType
         CN470RP1,
         CN470RP2,
         AS923,
+        AU915,
```
Comment (fbeltrao) [inline LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Regions/LoRaRegionType.cs:14]: Do we need to add the AU concentrator file to the lora-cli folder?

**Review by fbeltrao — APPROVED** [review]

File: LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWanNetworkSrvModule/Program.cs (hunk 1 above)
Comment (Mandur) [reply LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWanNetworkSrvModule/Program.cs:35]: No it was done in a previous PR where I stemmed this branch from, I need to roll it back

Comment (Mandur) [reply LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWanNetworkSrvModule/Program.cs:35]: Removed good catch! Thanks!

File: LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Regions/LoRaRegionType.cs (hunk 2 above)
Comment (Mandur) [reply LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Regions/LoRaRegionType.cs:14]: Great point, adding it
